addons/website_sale/static/src/js/website_sale.js

```diff
@@ -75,7 +75,9 @@ publicWidget.registry.websiteSaleCartLink = publicWidget.Widget.extend({
             self._popoverRPC = $.get("/shop/cart", {
                 type: 'popover',
             }).then(function (data) {
-                self.$el.data("bs.popover").config.content = data;
+                const popover = Popover.getInstance(self.$el[0]);
+                popover._config.content = data;
+                popover.setContent();
```

Comment:
If i'm not wrong it's more safe to set the `Tip` to ensure that it use the right `Tip` as the default value is `document.documentElement` (in `SelectorEngine.findOne()`).
```suggestion
                popover.setContent(popover.getTipElement());
```